bitnami/mariadb/10.9/debian-11/rootfs/opt/bitnami/scripts/libmariadb.sh

```diff
@@ -304,7 +304,8 @@ EOF
         if [[ -z "$DB_REPLICATION_MODE" ]] || [[ "$DB_REPLICATION_MODE" = "master" ]]; then
             if [[ "$DB_REPLICATION_MODE" = "master" ]]; then
                 debug "Starting replication"
-                echo "RESET MASTER;" | debug_execute "$DB_BIN_DIR/mysql" --defaults-file="$DB_CONF_FILE" -N -u root
+                [[ -f "$DB_EXTRA_CONF_FILE" ]] && DEFAULTS_EXTRA_FILE_OPT=--defaults-extra-file="$DB_EXTRA_CONF_FILE"
+                echo "RESET MASTER;" | debug_execute "$DB_BIN_DIR/mysql" --defaults-file="$DB_CONF_FILE" ${DEFAULTS_EXTRA_FILE_OPT:-} -N -u root
```

Comment:
To be consistent with the coding style, I would suggest using a `local` env variable instead of `DEFAULTS_EXTRA_FILE_OPT`:

```suggestion
               local -a flags=("--defaults-file=${DB_CONF_FILE}")
               [[ -f "$DB_EXTRA_CONF_FILE" ]] && flags+="--defaults-extra-file=${DB_EXTRA_CONF_FILE}"
                echo "RESET MASTER;" | debug_execute "$DB_BIN_DIR/mysql" "${flags[@]}" -N -u root
```